README.md

```diff
@@ -51,7 +94,7 @@ sam deploy \
 
 ## Testing the Example
 
-After you have created the stack using the CloudFormation template, you can test the system by uploading a Markdown file to the InputBucket that was created in the stack. You can use the sample-1.md and sample-2.md files in the repository as example files. After the files have been uploaded, you can see the resulting HTML and plain text files in the output bucket of your stack. You can also view the CloudWatch logs for each of the functions in order to see the details of their execution.
+After you have created the stack using the CloudFormation template, you can test the system by uploading a Markdown file to the InputBucket that was created in the stack. You can use the sample-1.md and sample-2.md files in the repository as example files. After the files have been uploaded, you can see the resulting HTML file in the output bucket of your stack. You can also view the CloudWatch logs for each of the functions in order to see the details of their execution.
```

Comment:
This is fine here but I'll likely be updating the directions for building, packaging, deploying, and testing in a subsequent PR.

Reply:
I don't expect this will be our last pass. We will need to review it again once we have all the relevant pieces in place.